src/SourceBuild/Arcade/eng/common/templates/steps/source-build-run-tarball-build.yml

```
@@ -19,7 +19,8 @@ steps:
 - script: |
     set -x
 
-    ./build.sh
+    ./build.sh \
+    --online
```

Additional indentation of the args would help the readability IMO.
```suggestion
      --online
```